sapmon/payload/sapmon.py

```diff
@@ -190,15 +176,16 @@ def addProvider(args: str = None,
    except Exception as e:
       tracer.critical("could not instantiate %s (%s)" % (providerType,
                                                          e))
-      return False
+      sys.exit(ERROR_ADDING_PROVIDER)
+
    if not newProviderInstance.validate():
       tracer.critical("validation check for provider instance %s failed" % newProviderInstance.fullName)
-      return False
+      sys.exit(ERROR_ADDING_PROVIDER)
    if not saveInstanceToConfig(instanceProperties):
       tracer.error("could not save provider instance %s to KeyVault" % newProviderInstance.fullName)
-      return False
+      sys.exit(ERROR_ADDING_PROVIDER)
    tracer.info("successfully added provider instance %s to KeyVault" % newProviderInstance.fullName)
-   return True
+
```

Comment:
I would suggest:
```suggestion
return
```
to denote the end of the method.

Reply:
Yah I decided to remove it to be consistent with the API design.